docs/authoring.md

```diff
@@ -162,14 +159,39 @@ Consuming C#/WinRT components from MSIX-packaged applications is supported with
 
 C#/WinRT supports authoring out-of-process components that can be consumed by Windows Runtime compatible languages. Currently, consuming an out-of-process component is supported for managed C# apps with the use of a packaging project. This scenario requires creating a `WinRT.Host.runtimeconfig.json` file, as demonstrated in the [background task component sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/BgTaskComponent). Native C++ consumption of out-of-process C# components is not fully supported yet.
 
+### Windows App SDK Applications
+
+Authoring WinUI types with C#/WinRT is in development and is supported with some exceptions.
+Component authors can implement WinUI types using the Windows App SDK class library template. 
+Components can be consumed by application projects that reference the Windows App SDK.
+
+When referencing a C# authored component via project reference, the following application types are not supported currently:

```

Comment:
from reading this I know what _not_ to do but not what to do. Can we also list which configuration _is_ supported?

Reply:
thanks! sure, added

Reply:
shoudl we list what is supported first?

Reply:
Sure!